src/logger/log_entry.py

```diff
@@ -14,15 +23,16 @@ class Schema(BaseModel.Schema):
         """The marshmallow schema for the LogEntry model."""
 
         timestamp = marsh.fields.DateTime(required=True)
+        tick = marsh.fields.Integer(required=True)
         message = marsh.fields.String(required=True)
         run_id = marsh.fields.UUID(required=True)
 
         def _make(self, data: dict) -> "LogEntry":
             return LogEntry(**data)
 
     timestamp = DateTimeField(null=False, default=datetime.now())
-
-    message = CharField(null=False)
+    tick = IntegerField(null=False)
```

Comment:
Do you think integer is enough? 🤔 To get meaningful data, we need to run the simulation for some time.

Reply:
I don't know what you mean exactly. In python integers are unbound and in PostgreSQL e.g. the limit is at 2147483647, which should be plenty to run some simulations if we assumed that every tick more or less represents a second in the simulation.

Reply:
Sorry that I only wrote such a short description 😄 
I meant that we must have long simulation runs to collect meaningful data. That means the ticks shouldn't limit us. If somebody wants to simulate over 70 years, then the person shouldn't be restricted. As developers, it's not a big deal to use BigInteger Fields in the database. But for the user, it could be a pain.
That's my opinion :D I understand the point that nobody would simulate over 70 years 😂 
You decide 😇 It was just an idea.

Reply:
I mean, it is no problem to change it, it might be good in the future to have this possibility

Reply:
Should be changed now, I'm assuming that the Marshmallow integer field corresponds with the Python integer field, in that they both allow for unbound values